lib/pengine/unpack.c

```diff
@@ -2828,7 +2831,7 @@ unpack_migrate_from_failure(pe_resource_t *rsc, pe_node_t *node,
         pe_node_t *source_node = pe_find_node(data_set->nodes, source);
 
         if (source_node && source_node->details->online) {
-            native_add_running(rsc, source_node, data_set);
+            native_add_running(rsc, source_node, data_set, action_fail_ignore);
```

Comment:
This is a failure as well, so I think some other action_fail_ makes sense

Reply:
Okay!

Set failures other than action_fail_ignore.